### DISCUSSION TITLE
ADD THIS EXAMPLE

### DISCUSSION THREAD

**Discussion by jogsg** [ntsc-rs/ntsc-rs#153] [Ideas]
https://www.youtube.com/watch?v=nqy_hYDI0As
THIS BECAUSE VHS COPIED TO VHS TO VHS TO VHS GOTTA GENERATE IT

**Comment by Degamisu** [comment]
VHS generation loss was mentioned a long time ago and @valadaptive said something about it not being planned. I can't find the original issue, but it said something along the lines of:

> This can already be done by just rendering the video and putting it back into the effect

While this was a good idea, it can easily be replicated by adjusting the bandwidth.

Hope this helps!

>[!NOTE]
>All the suggestions can go in the [issues tab](https://github.com/valadaptive/ntsc-rs/issues) as the `discussions` tab is not really read often!

**Comment by jogsg** [comment]
yeah but it little sucks it not like real vhs

**Reply by Degamisu** [reply]
Well, it technically is. I think it runs off of generated signals. If you'd like, I have some examples for you!

**Reply by jogsg** [reply]
sure gift it to me

**Reply by Degamisu** [reply]
![image](https://github.com/user-attachments/assets/5d4a05e3-5be9-444f-9606-db7373bb7097)
© @rekkol

![image](https://github.com/user-attachments/assets/9900989c-cdd5-48a6-8149-a0abdaf2030d)
© @Degamisu

**Comment by jogsg** [comment]
also the audio gotta be glitched

**Reply by Degamisu** [reply]
Audio distortion was mentioned here. #15
